packages/edit-post/src/components/header/writing-menu/index.js

```diff
@@ -37,11 +76,21 @@ function WritingMenu() {
 				scope="core/edit-post"
 				name="fullscreenMode"
 				label={ __( 'Fullscreen mode' ) }
-				info={ __( 'Work without distraction' ) }
+				info={ __( 'Show and hide admin UI' ) }
 				messageActivated={ __( 'Fullscreen mode activated' ) }
 				messageDeactivated={ __( 'Fullscreen mode deactivated' ) }
 				shortcut={ displayShortcut.secondary( 'f' ) }
 			/>
+			<PreferenceToggleMenuItem
+				scope="core/edit-post"
+				name="distractionFree"
+				toggleHandler={ toggleDistractionFree }
+				label={ __( 'Distraction free' ) }
+				info={ __( 'Write with calmness' ) }
+				messageActivated={ __( 'Distraction free mode activated' ) }
```

Comment:
These messages only appear for me when I use the keyboard shortcuts, not toggling the button. I'm not sure if that's the expected behaviour.

Reply:
They're the same text but triggered by different things. In the case of DFM the keybiard shortcut handler triggers the message. I think it's good as it stops you from panicking you did something wrong :). We can revise it.